components/Tabs/Tabs.js

```diff
@@ -1,53 +1,43 @@
 
 class TabLink {
-  constructor(element) {
-    // Assign this.element to the passed in DOM element
-    // this.element;
-    
-    // Get the custom data attribute on the Link
-    // this.data;
-    
-    // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
-    // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
-    // Add a click event listener on this instance, calling the select method on click
-
-  };
-
-  select() {
-    // Get all of the elements with the tabs-link class
-    // const links;
-
-    // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
-    // Add a class named "tabs-link-selected" to this link
-    // this.element;
-    
-    // Call the select method on the item associated with this link
+  constructor(link) {
+    this.link = link;
+    //console.log(this.link.dataset.tab);
+    this.data = document.querySelector(`.tabs-item[data-tab="${this.link.dataset.tab}"]`); 
+    this.itemElement = document.querySelector(`.tabs-item[data-tab='${this.data}']`);
+    this.tabItem = new TabItem(this.data);
+    this.link.addEventListener('click', this.select.bind(this));
 
   }
+  select() {  
+    //// Get all of the elements with the tabs-link class
+    const links = document.querySelectorAll('.tabs-link');
+    //// Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
+    links.forEach(link => {
+      link.classList.remove('tabs-link-selected');
+    })
+    //// Add a class named "tabs-link-selected" to this link
+    this.link.classList.add('tabs-link-selected');
+    this.tabItem.select();
+  };
 }
 
 class TabItem {
-  constructor(element) {
-    // Assign this.element to the passed in element
-    // this.element;
+  constructor(tabItem) {
+    //// Assign this.element to the passed in element
+    this.tabItem = tabItem;
   }
-
   select() {
-    // Select all ".tabs-item" elements from the DOM
-    // const items;
-
-    // Remove the class "tabs-item-selected" from each element
-    
-    // Add a class named "tabs-item-selected" to this element
-    //this.element;
+    //// Select all ".tabs-item" elements from the DOM
+    const items = document.querySelectorAll('.tabs-item');
+    //// Remove the class "tabs-item-selected" from each element
+    items.forEach(tabItem => tabItem.classList.remove('tabs-item-selected'));
+    //// Add a class named "tabs-item-selected" to this element
+    //console.log(this.tabItem);
+    this.tabItem.classList.add('tabs-item-selected');
+    // this.tabItem.select();
   }
-}
+};
```

Comment:
You can remove the line comment here in line 38, since it does not document anything anyway.